src/index.php

```diff
@@ -26,6 +26,12 @@
 // get user from url query string
 $user = $_REQUEST["user"];
 
+// redirect to sample site
+if(!isset($_REQUEST["user"]){
+    header('Location: https://github-readme-streak-stats.herokuapp.com/demo');
```

Comment:
`demo/` is a more versatile solution since it will work on all deployments no matter where it is hosted

```suggestion
    header('Location: demo/');
```